src/pgw_s5s8.erl

```diff
@@ -436,7 +444,8 @@ handle_request(ReqKey,
     ResponseIEs = create_session_response(ActiveSessionOpts, IEs, EBI, Context),
     Response = response(create_session_response, Context, ResponseIEs, Request),
     gtp_context:send_response(ReqKey, Request, Response),
-    {keep_state, Data#{context => Context, pfcp => PCtx, pcc => PCC4}};
+    {keep_state, Data#{context => Context, pfcp => PCtx, pcc => PCC4},
+     [{{timeout, gtp_idle_timer}, Timeout, stop_session}]};
```

Comment:
a wrapper function the generate the action might make the code more readable
I'm thinking about something like:

```
context_idle_action(Actions, #{timeout := Timeout})
  when is_integer(Timeout) ->
    [{timeout, context_idle}, Timeout, stop_session}}|Actions];
context_idle_action(Actions, _) ->
    Actions.
```

And then use that, e.g.
```
  Actions = context_idle_action([], Data),
  {keep_state, ..., Actions}
```